main.js

```diff
@@ -140,8 +132,9 @@ export function getCredentialStatus({credential, statusPurpose} = {}) {
   const credentialStatuses = _getStatuses({credential});
   const result = credentialStatuses.filter(
     credentialStatus => _validateStatus({credentialStatus})).find(
-    // check for matching `type` (legacy) or `statusPurpose`
-    cs => cs.type === expectedType || cs.statusPurpose === statusPurpose);
+    // check for matching `type` or `statusPurpose`
+    cs => cs.type === 'StatusList2021Entry' ||
+      cs.statusPurpose === statusPurpose);
```

Comment:
Why is this still an `or` condition now that the legacy has been removed? Seems like this logic needs to be made consistent with the data format and how this particular function works.

Reply:
Updated [11db1a3](https://github.com/digitalbazaar/vc-status-list/pull/6/commits/11db1a35e3a6c9aa948dc63b06d6159692cd79b4)